src/react/components/common/tagInput/tagInput.scss

```diff
@@ -0,0 +1,126 @@
+@import "../../../../index.scss";
+
+$tagInputWidth: 250px;
+
+.condensed-list-header {
+    display: flex;
+    flex-direction: row;
+}
+
+.condensed-list-title {
+    flex: 1;
+}
+
+.tag {
+    &-input {
+        background: $lighter-1;
+        min-width: $tagInputWidth;
+        
+        &-text-input-row {
+            padding: 2px 0;
+
+            input {
+                min-width: $tagInputWidth;
```

Comment:
Any reason not to use 100% here? So it will just fill up the available space? Or use flexbox and setup flex-grow.